BitwardenShared/UI/Autofill/Utilities/Fido2UserVerificationMediatorTests.swift

```diff
@@ -268,6 +286,39 @@ class Fido2UserVerificationMediatorTests: BitwardenTestCase {
                 credential: cipher
             )
         }
+        XCTAssertTrue(fido2UserVerificationMediatorDelegate.onNeedsUserInteractionCalled)
+    }
+
+    private func checkUser_throws_when_repromptNeedsUserInteraction(
```

Comment:
🎨 When there are internal functions that perform testing things and can fail a test on an assertion, I like taking advantage of the [file and line parameters of the XCTAssert varieties](https://developer.apple.com/documentation/xctest/1500669-xctassert), and in particular setting it up to use the file and line of the test that's calling the function, similar to what's discussed [here](https://www.iosdev.recipes/xctest/custom-assertions-file-and-line/) but frequently also with custom failure text. This means that when a test fails, the failure shows in Xcode in the test where it fails, so there's appropriate context to understand what's going on; this is particularly useful if there are several different parameters that cause a failure, because without it Xcode logs all of the errors inside of the helper function, which is not necessarily helpful for diagnosing what conditions led to the failure and why.

It does look like modern Xcode ameliorates this a bit, so it's probably a moot point, but I still think it would be a good pattern to consider adopting.

Reply:
Yeah, sounds good I'll add it 😄